rebar.config

```diff
@@ -17,7 +17,8 @@
   {rebar3_format, "~> 1.2.0"},
   {rebar3_lint, "~> 1.0.2"},
   {rebar3_sheldon, "~> 0.4.2"},
-  {rebar3_ex_doc, "~> 0.2.9"}]}.
+  {rebar3_ex_doc, "~> 0.2.9"},
+  {jsx, "~> 2.10"}]}.
```

Comment:
Why not go with 3.x, here? I remember having to patch a bunch of code with the `return_maps` flag to avoid incompatibility...